### PR TITLE
add Fhem LXC script

### PR DESCRIPTION
**Pull request by samydoesit**
## Description

added Fhem LXC script (https://fhem.de/)

## Type of change

- [x] New Script

### CODE REVIEW

File: ct/fhem.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+header_info
+echo -e "Loading..."
+APP="Fhem"
+var_disk="8"
+var_cpu="2"
+var_ram="2048"
+var_os="debian"
+var_version="11"
+# var_os="ubuntu"
```
Comment (tteck) [inline ct/fhem.sh]: remove `# var_os="ubuntu"` and `# var_version="22.04"`

File: install/fhem-install.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+$STD apt-get install -y mc
+$STD apt-get install -y avahi-daemon
+$STD apt-get install -y gnupg2
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up Fhem Repository"
+wget -O- https://debian.fhem.de/archive.key | gpg --dearmor | tee /usr/share/keyrings/debianfhemde-archive-keyring.gpg > /dev/null
+sh -c 'echo "deb [signed-by=/usr/share/keyrings/debianfhemde-archive-keyring.gpg] https://debian.fhem.de/nightly/ /" >> /etc/apt/sources.list'
+msg_ok "Set up Fhem Repository"
+
```
Comment (tteck) [inline install/fhem-install.sh:28]:
```
msg_info "Setting up Fhem Repository"
curl -sSf https://debian.fhem.de/archive.key | gpg --dearmor >/etc/apt/trusted.gpg.d/debianfhemde-archive-keyring.gpg
echo 'deb [signed-by=/etc/apt/trusted.gpg.d/debianfhemde-archive-keyring.gpg] https://debian.fhem.de/nightly/ /' >/etc/apt/sources.list.d/fhem.list
msg_ok "Set up Fhem Repository"
```

File: install/fhem-install.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+msg_info "Installing Fhem"
+$STD apt-get update
+$STD apt-get install fhem
```
Comment (tteck) [inline install/fhem-install.sh]: `apt-get install -y fhem`

**Comment by tteck** [comment]
Thanks! I'll add it to gh-pages.

**Comment by samydoesit** [comment]
Thank you. 🚀

Could you please add the hint regarding USB passthrough to gh-pages, stating that if the LXC is created Privileged, the script will automatically set up USB passthrough? 🙂

**Comment by tteck** [comment]
It's my belief that many people are already aware that any LXC created from this repository will enable USB passthrough automatically when created with privileged access. In case you prefer this setting, you can change the default value of `CT_TYPE` from `1` to `0` in the `fhem.sh` and submit a pull request.

**Comment by tteck** [comment]
Added: "🛈 If the LXC is created Privileged, the script will automatically set up USB passthrough."

**Pull request merged**